autotst/calculator/vibrational_analysis.py

```diff
@@ -214,11 +226,21 @@ def validate_ts(self):
 
             self.percent_changes = self.obtain_percent_changes()
 
+            if (self.percent_changes[self.percent_changes.center].percent_change == 0.0).any():
+                logging.error('Distance between reacting atoms is too big, we cannot validate this TS through vibrational analysis.')
+                return False
+            
+            center_mean = self.percent_changes[self.percent_changes.center].percent_change.mean()
+            shell_mean = self.percent_changes[self.percent_changes.center == False].percent_change.mean()
+            if center_mean < 15:
```

Comment:
Is the 15% arbitrary?  Any data to justify this cutoff?  Perhaps we should be using + and - the negative freq vibrational displacements when calculating the bond distance (TS + disp vs TS - disp, instead of TS vs TS + dis).  That way, the percent change in reaction center bonds will be greater for valid TS.  Would still need the optimized TS geom for checking the reference distance

Reply:
Okay, what we see here are vibrational analysis results for about 1750 TS geometries run with the current vibrational analysis workflow *BUT* have the d12 / d13 / d23 distance requirement. After seeing this, I'm tempted to say above 25% just to be sure. Thoughts?

![Screen Shot 2020-05-14 at 1 21 44 PM](https://user-images.githubusercontent.com/20301547/81965317-e1689b80-95e5-11ea-876a-d73cd964c6bc.png)

Reply:
Yes, based on this, 25% looks good.  For the few cases where the center is <25%, do you know if those are valid TSs?